cns/saas/admin_guide/start/enforcer/enforcer.adoc

```diff
@@ -1,26 +1,297 @@
+
+[.procedure]
+
+. Obtain your GCP project ID.
+
+
+. On Prisma Cloud, navigate to the *Namespace* of the group where you want to deploy the Enforcer.
+. Click on the *Authorizations* tab.
+. Click on the `+` button.
+. Select *Create a cloud auto-registration policy*.
++
+image::cloud-registration-1.png[width=250,align="center"]
+
+. Go to *Auto-registration*
+. Under *Cloud Provider* select *GCP*
+. Under *Claims*, enter your project ID, for example `projectid=<your GCP project ID>`
+. Click *Next* to finish creating the auto-registration policy.
++
+image::cloud-registration-4.png[width=300,align="center"]
+
+==== Create an Auto-Registration Policy using the CLI
+
+To include the creation of the auto-registration policy in your automation, you can use a single command using the `apoctl` command-line utility.
+
+Use the following example to create an auto-registration policy. Replace the `<placeholders>` with the appropriate values for your CSP and environment.
+
+[source]
+----
+apoctl api create apiauth \
+ -n '<namespace>' \
+ -k name '<cloud autoregistration name>' \
+ -k subject '[["@auth:realm=<source>", "@auth:<CSP account>"]]' \
+ -k authorizedNamespace '<namespace>' \
+ -k authorizedIdentities '["@auth:role=enforcer"]'
+----
+
+[.task]
+==== Linux
+
+Before you begin, ensure that an auto-registration policy already exists.
+
+[.procedure]
+
+. Navigate to the *Namespace* of the group where you want to deploy the Enforcer. From there, go to the *Agent > Deploy* page.
+
+. Select *Single Enforcer*.
+
+. Set the *Platform Type* to *Linux*.
+
+. Set the *Registration Type* to *Cloud Autoregistration*.
++
+image::install-enforcer-6.png[width=300,align="center"]
+
+. Copy the resulting installation script and run it on your VMs to install the Enforcer.
++
+image::install-enforcer-7.png[width=300,align="center"]
+
+[.task]
+==== Windows
+
+Before you begin, ensure that an auto-registration policy already exists.
+
+[.procedure]
+
+. Navigate to the *Namespace* of the group where you want to deploy the Enforcer. From there, go to the *Agent > Deploy* page.
+
+. Select *Single Enforcer*.
+
+. Set the *Platform Type* to *Windows*.
+
+. Set the *Registration Type* to *Cloud Autoregistration*.
++
+image::install-enforcer-9.png[width=300,align="center"]
+
+. Copy the resulting installation script and run it on your clients and servers to install the Enforcer.
++
+image::install-enforcer-10.png[width=300,align="center"]
+
+
+
+=== Enforcer Tags
+
+Enforcer tags are used when you want to create tags for the Enforcer itself.
+This capability is very useful on environments where security administrators have no permissions to create or modify existing workload tags.
+
+In such cases, administrators can use Enforcer tags as a way to use custom tags on rulesets, and then reference the tags when installing an Enforcer.
+
+image::install-enforcer-2.png[width=300,align="center"]
+
+=== Install the Kubernetes CRD Operator
+
```

Comment:
Kubernetes CRD Operator allows kubernetes operators to use the `kubectl` cli to natively manage the CNS objects. This removes the need for `apoctl` cli. The operator uses the namespace secret as token to communicate to the CNS API server, hence you don't need app creds. This pattern is useful for automation, where you can remove this dependency on the `apoctl` cli.  For all the functionality of the CRD, refer to the CRD doc page.

Reply:
Thank you for providing the additional context @rjain15 I've added the information but did not include the last bit about CRD information. Information about CRDs is outside of the scope of this doc set and readily available in the K8s docs.